CHANGELOG.md

```diff
@@ -8,8 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 ### Changed
+- Return ResponseInterface in controller actions (#597)
```

Comment:
And to make the changelog even more beautiful, please put code, class names etc. (in this case, `ResponseInterface`) in backticks so it gets rendered in a monospace font. (GitHub parses Markdown in the release notes, too.)